Firestore/core/src/local/local_store.cc

```diff
@@ -76,6 +79,21 @@ using remote::TargetChange;
  */
 const int64_t kResumeTokenMaxAgeSeconds = 5 * 60;  // 5 minutes
 
+DocumentKeySet GetKeysWithTransformResults(
+    const MutationBatchResult& batch_result) {
+  DocumentKeySet result;
+
+  for (size_t i = 0; i < batch_result.mutation_results().size(); ++i) {
+    if (batch_result.mutation_results()[i]
```

Comment:
nit: Consider saving `batch_result.mutation_results()[i]` to a local variable to avoid retrieving it twice, and improving readability a little bit.